data/layout/telescope_positions-North-1SST.ecsv

```diff
@@ -52,4 +52,4 @@
 #       value: 3.0
 # schema: astropy-2.0
 telescope_name pos_x pos_y pos_z lat lon utm_east utm_north alt
-S-01 0.0 0.0 0.0 28.762166100000005 -17.8920302 217609.2270142641 3185067.278324084 2154.75
+S-01 0.0 0.0 0.0 28.762158000000007 -17.891571 217654.06411838497 3185065.2899643113 2154.75
```

Comment:
Having a small-sized telescopes in the north will raise questions.

Reply:
I created all the possibilities of 1 and 4 telescope arrays, as is done in sim_telarray. Better remove the North-1SST then?

Reply:
There will be no SSTs in the north, so that is not needed.